client/components/user-home.js

```diff
@@ -7,19 +7,20 @@ import Map from './Map';
 import List from './List';
 import Filter from './Filter';
 import { Route } from 'react-router-dom';
+// import { withRouter, Link } from 'react-router-dom'
 import reportForm from './report-form';
 import PointsMeter from './pointsmeter';
+import Profile from './profile';
 import {Grid, Row, Col} from 'react-bootstrap';
 
-export class UserHome extends Component {
 
+export class UserHome extends Component {
   constructor() {
     super();
     this.state = {
-      // currentLong: 0,  // this two might not be neccessary
-      // currentLat: 0,   // this might not be neccessary
-      showNotification: {isShow: false, message: ''},
```

Comment:
Was this line deleted on purpose or by accident? Just not sure if its needed still (@timkao )